app/script/view_model/content/InputBarViewModel.js

```diff
@@ -227,6 +194,22 @@ z.viewModel.content.InputBarViewModel = class InputBarViewModel {
       return conversationEntity.localMessageTimer() && !conversationEntity.hasGlobalMessageTimer();
     });
 
+    this.editedMessage = ko
+      .pureComputed(() => {
+        const text = this.input();
+        const mentions = this.currentMentions();
+        return {mentions, text};
+      })
+      .extend({rateLimit: {method: 'notifyWhenChangesStop', timeout: 1}});
+
+    this.conversationEntity.subscribe(this.loadInitialStateForConversation.bind(this));
+    this.editedMessage.subscribe(message => {
```

Comment:
I consider this naming confusing. The `editedMessage` calls `_saveEditState` to persist the state but not if `isEditing` is true. We need to clearly seperate the naming regarding editing an existing message (changing it) and drafting a new message.

Reply:
Oh true. `draft` sounds great actually, I would go for `draftMessage`, WDYT?